frontend/src/components/tables/DataTableHebrew.tsx

```diff
@@ -100,15 +103,18 @@ export function DataTableHebrew<TData, TValue>({
       <div className="flex flex-col ">
         <div className="flex items-center py-4 gap-4">
           <Input
-            placeholder="חיפוס..."
```

Comment:
this code works for us but takes away from the genericness (There def is a better way to say that) of the table. 

Now if i pass something that does not have the firstName accessor key it will not work. Add User is also a very specific to a user table. Add item would be better. Obviously we want it to say user. Maybe you can do something to make it dynamic.